app/scripts/controllers/overview.js

```diff
@@ -1090,6 +1092,10 @@ function OverviewController($scope,
     updateFilter();
   });
 
+  $scope.browseCatalog = function() {
+    $location.url('project/' + $scope.projectName + '/project-browse-catalog');
```

Comment:
Suggest adding this to the `Navigate` service so we only ever have one place where we build the URL.